src/analysis/complex/exponential.lean

```diff
@@ -1771,6 +1782,10 @@ continuous_within_at.continuous_at
   (continuous_on_iff_continuous_restrict.2 continuous_rpow_aux3 _ hy)
   (mem_nhds_sets (by { convert is_open_prod is_open_univ (is_open_lt' (0:ℝ)), ext, finish }) hy)
 
+lemma continuous_at_rpow {x y : ℝ} (h : x ≠ 0 ∨ y > 0) :
```

Comment:
```suggestion
lemma continuous_at_rpow {x y : ℝ} (h : x ≠ 0 ∨ 0 < y) :
```

Reply:
Indeed, what was I thinking. It must be because I have a bad flu (but very likely only a flu).